### PR TITLE
Sync new "started sign up" event to emailers.

### PR DESCRIPTION
**Pull request by alanpeabody**

### CODE REVIEW

File: spec/ello/kinesis_consumer/mailchimp_processor_spec.rb [hunk 1]
```diff
@@ -53,6 +53,37 @@
       end
     end
 
+    describe 'when presented with a Send event' do
```
Comment (Justin-Holmes) [inline spec/ello/kinesis_consumer/mailchimp_processor_spec.rb]: Pretty minor, but it should probably be `StartedSignUp` event here.

**Comment by Justin-Holmes** [comment]
:metal:

File: lib/ello/kinesis_consumer/knowtify_processor.rb [hunk 2]
```diff
@@ -19,6 +19,20 @@ def invitation_was_sent(record)
                                 }]
       end
 
+      def started_sign_up(record)
+        knowtify_client.upsert [{ email: record['email'],
+                                  data: {
+                                    subscribed_to_users_email_list: record['subscription_preferences']['users_email_list'],
```
Comment (jayzes) [inline lib/ello/kinesis_consumer/knowtify_processor.rb]: Would we want these to explicitly be false to start?

Comment (alanpeabody) [inline lib/ello/kinesis_consumer/knowtify_processor.rb]: So I followed the same pattern as with invites, which basically is that the mothership is the authoritative source of these, where they are explicitly set to true.

Comment (alanpeabody) [inline lib/ello/kinesis_consumer/knowtify_processor.rb]: Also, I don't know if the value should actually be true or false, but as this was replacing the "system generated invitation" we assumed we should subscribe the same way as before.

Comment (jayzes) [inline lib/ello/kinesis_consumer/knowtify_processor.rb]: Ah, right, we explicitly wanted the flow to be that you get on the lists as soon as you get an invite/provide your e-mail. Nevermind then.

**Comment by alanpeabody** [comment]
We have not tested or merged the mothership for this but I am going to merge to avoid conflicts.

**Pull request merged**